platform/ui/src/components/SidePanel/SidePanel.tsx

```diff
@@ -5,6 +5,7 @@ import { useTranslation } from 'react-i18next';
 
 import Icon from '../Icon';
 import Tooltip from '../Tooltip';
+import { Separator, ToggleGroup, ToggleGroupItem } from '@ohif/ui-next';
```

Comment:
This seems unusual. All of this should move to ui-next. There shouldn't be any imports here. Ideally, this PR shouldn't touch platform/ui at all. Any necessary code should be copied and pasted into ui-next, as we'll eventually delete platform/ui.

Reply:
good callout, This flew over my head at the start, now it makes sense